dist/snapcraft.yaml

```diff
@@ -0,0 +1,123 @@
+##########
+# Freeciv21 - snapcraft.yaml
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2023 James Robertson <jwrober@gmail.com>
+#
+# Build Steps:
+#   - (once) sudo snap install snapcraft --classic
+#   - (once) sudo snap install multipass
+#   - mkdir -p <source_dir>/build/snap/local
+#   - cp <source_dir>/dist/snapcraft.yaml <source_dir>/build/snap
+#   - cp <source_dir>/data/icons/128x128/freeciv21-client.png <source_dir>/build/snap/local
+#   - cd <source_dir>/build
+#   - snapcraft
+#   - sudo snap install --devmode ./freeciv21_*.snap
+#
+##########
+
+name: freeciv21
+title: Freeciv21
+summary: Freeciv21 - Freeciv for the 21st Century
+description: |
+  Freeciv21 is a free and open source empire-building strategy game inspired by the history of human civilization. The game commences in prehistory and your mission is to lead your tribe from the Stone Age to the Space Age.
+  This snap package contains everything you need to play local single-player games against the computer as well as connect to online multi-player games against human opponents.
+  Freeciv21 is based on the Qt framework and brought to you by the team at Longturn.net.
+
+# TODO: At some point, will want to integrate into cmake process and configure_file()
+version: '3.0-rc.1'
```

Comment:
Let's make an issue for this right away.

Reply:
#1735